### PR TITLE
Portfolio Project Milestone 2

### PR DESCRIPTION
**Pull request by Santosh-Konappanavar**
Portfolio Project Milestone 2
The following features have been added to the portfolio website.

The Portfolio website now has the details of each project.
The about me page was added to provide a brief description of my experience
The languages, frameworks and skills have also been added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,5 +53,144 @@ <h2 class=" role"> I am a Software developer</h2>
         
     </div>
     
+    <section class ="worksection">
```
Comment (MeqdamAlqudah) [inline index.html:56]: - [ ] Please make sure that the mrgin between the paragraph and the tech list matches the design 
your code 
image
Figma 
image

- [ ] Please fix the margin between the work section and the about me section
image
the design 
image

File: index.html [hunk 2]
```diff
@@ -53,5 +53,144 @@ <h2 class=" role"> I am a Software developer</h2>
...
+                        <li class="list">Github</li>
+                        <li class="list">Codepen</li>
+                        <li class="list">Gitlab</li>
+                        <li class="list">Terminal</li>
+                    </ul>
+                    
+                </li>
+            </ul>
+
+        </div>
```
Comment (MeqdamAlqudah) [inline index.html:191]: - [ ] Please fix the margin between the about me paragraph and the button here 
image

File: index.html [hunk 3]
```diff
@@ -53,5 +53,144 @@ <h2 class=" role"> I am a Software developer</h2>
...
+
```
Comment (MeqdamAlqudah) [inline index.html:192]: - [ ] Please make sure that the font family matches the design 
image
Figma design 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Santosh-Konappanavar  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
image


 ## Highlights
:+1: Readme file
:+1: no Linter errors
:+1:  you got most of the project from the first time

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @Santosh-Konappanavar  👋 
 
 # STATUS: APPROVED ✔️  🍾  🥳

 Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉

 
 
 ## Highlights
 - Great design 👏 
 -  Descriptive Pull Request(PR) ✅ 
- Proper use of Github features
 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 

 Cheers and Happy coding!👏👏👏
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**